.github/workflows/build-test-distribute.yml

```
@@ -34,7 +34,7 @@ jobs:
 
   ubuntu22-build-test:
     needs: [ prepare-image ]
-    uses: MeshInspector/MeshLib/.github/workflows/build-test-ubuntu22.yml@master
+    uses: MeshInspector/MeshLib/.github/workflows/build-test-ubuntu22.yml@ubuntu22-gcc12
```

Undo debug before merging
```suggestion
    uses: MeshInspector/MeshLib/.github/workflows/build-test-ubuntu22.yml@master
```